setup.py

```diff
@@ -69,6 +69,8 @@
                'prettytable',
                'tifffile>=2018.10.18',
                'numba',
+                # included in stdlib since v3.8, but this required version requires Python 3.9
+               'importlib_metadata>=1.6.0',
```

Comment:
```suggestion
               'importlib_metadata>=1.6.0;python_version<"3.9.X"',
```
In case you know which minimum version of python include `importlib_metadata`, it may be good to specify it, to make it easier in the future to know when to remove it?

Reply:
At the moment their API is a bit unstable, so I think that it is good to use the package version regardless of the Python version. In this way we can better manage the required version. In the future I imagine that the package version will be deprecated so  the deprecation warning should be enough to remind us this. Does it make sense?

Reply:
yes, fine, there is already a comment.